test/keyword_type_test.js

```diff
@@ -19,14 +18,16 @@ describe('KeywordType Controller', function() {
     beforeEach('Populate SQLite memory DB with fresh test data', function() {
         return knex.seed.run();
     });
-    describe('GET Keyword Type', function(){
+    describe('GET /api/keywordtypes/', function(){
+
         it('Successfully retrieves KeywordType by id in the database', function(done) {
-            let testdata=testdata.keyword_types[0];
+            let testKeywordTypes = testdata.keyword_types;
+            
             chai.request(server)
-                .get(testdata.id)
+                .get('/api/keywordtypes/' + testKeywordTypes)
```

Comment:
Two things: 
1. keywordtype, not keywordtypes (See above)
2. You are appending an array to a string as part of the endpoint. The string ends up looking something like this: "/api/keywordtypes/[Array]". This is where your 404 is coming from. It should just be '/api/keywordtypes/'.